conf/rconfc_link_fom.c

```diff
@@ -179,6 +179,7 @@ static void rconfc_herd_link_fom_wait_on(struct rconfc_link *lnk,
 		[M0_RLF_SESS_TERMINATING] = rconfc_link__on_conn_terminated,
 	};
 
+	m0_clink_cleanup(&lnk->rl_fom_clink);
```

Comment:
Can we add a comment, the reason being doing a cleanup before init.

Reply:
update comment